app/views/docs/migrations-firebase.phtml

```diff
@@ -0,0 +1,173 @@
+<p>
+  Moving your project from Firebase to Appwrite?
+  Appwrite Migrations can help you streamline the process.
+
+  Here's what you need to know to get started.
+</p>
+
+<h2><a href="#notices" id="notices">Things to keep in mind</a></h2>
+
+<ol class="margin-top margin-bottom-large text-size-normal">
+  <li>Appwrite will not incur usage charges during migrations, but Firebase may still incur service charges.</li>
+  <li>Appwrite Migrations only supports Firestore as a database source. Realtime Database is <b>currently not supported</b>.</li>
+  <li>At the moment only top level document migration is supported. Nested documents will not be transferred automatically.</li>
+  <li>OAuth users will not be transferred. Users will need to re-authenticate with your OAuth provider after the migration is complete.</li>
+</ol>
+
+<h2><a href="#migration-process" id="migration-process">Migrating to Appwrite from Firebase</a></h2>
+<p>
+  To begin migrating to Appwrite make sure to read the <a href="/docs/migrations">migration overview</a>
+  and <a href="#notices">things to keep in mind</a> sections above.
+</p>
+
+<ol class="margin-top margin-bottom-large text-size-normal">
+  <li>
+    Create a new project and click on the <b>Migrations</b> tab.
+  </li>
+  <li>
+    Click on the <b>Create Migration</b> button and select <b>Firebase</b> as your source.
+  </li>
+  <li>
+    Depending on where your Appwrite project is hosted, you'll need to use different methods to authorize with Firebase.
+    See the <a href="#cloud">Cloud</a> and <a href="#self-hosted">Self-hosting</a> sections for details.
+  </li>
+  <li>
+    Finally, add the platforms for your <a href="/docs/getting-started-for-web">Web</a>, <a href="getting-started-for-flutter/docs/getting-started-for-flutter">Flutter</a>, <a href="/docs/getting-started-for-android">Android</a>, and <a href="/docs/getting-started-for-apple">iOS</a> apps.
```

Comment:
We dont mention this step in NHost, Supabase, ... We should either mention it everywhere, or nowhere.